### ISSUE TITLE
Bug: Keyboard Users should be able to scroll content inside of a tip manager

### ISSUE THREAD

**Issue by annelfitz** [Esri/calcite-app-components#306]
### Summary
When tabbing through the tip manager, the user should be able to use the up and down arrow keys to scroll through the content of the tip manager. Currently when the overall div of the tip manager is in focus, the up and down and left and right buttons navigate the user through each of the tips, when it should support scrolling. The user should just be able to navigate through the tips from the arrow keys at the bottom of the tip manager.

### Actual Behavior
When using the arrow keys when the tip manager is focused, you are navigated through each of the tips.

### Expected Behavior
When using the arrow keys when the tip manager is focused, you should be able to scroll through the content of a tip.

### Reproduction Steps

1. https://esri.github.io/calcite-app-components/demos/calcite-tip-manager.html
2. Tab through the page to highlight the tipmanager, then use the arrow keys. Will navigate through the tips instead of scrolling through the content

**Comment by driskull** [comment]
I can take this

**Comment by annelfitz** [comment]
Just tested locally and the scroll is still not working where I would expect. When the entire tip manager is highlighted, that's when I would expect to be able to use the arrows to scroll through the content, but this is still not possible. Now it is possible to scroll once the link inside the tip is in focus, but I think the scrolling should be set when the focus is around the whole text, not just a few words
![calicite_tipmanager_scroll](https://user-images.githubusercontent.com/51344676/66140467-06f4e780-e5b7-11e9-82f3-d902a5b4bd18.gif)

**Comment by annelfitz** [comment]
Ok I spoke too soon - this works in Firefox. A highlight appears around the content of the tip and scrolling is good! Not sure why it wasn't working in chrome ^^^
![Screen Shot 2019-10-03 at 8 30 29 AM](https://user-images.githubusercontent.com/51344676/66141195-26d8db00-e5b8-11e9-8477-87c77b41daf1.png)

**Comment by driskull** [comment]
> I think the scrolling should be set when the focus is around the whole text

This isn't something we have control of. The scrollable node is just the one inside the tip manager.

**Comment by annelfitz** [comment]
@driskull In Firefox, a focus outline appears around the scrollable content. Is there a way to do this for the other browsers? Currently (in all browsers but Firefox) the focus outline skips from the X of the tip manager to the link inside of the tip content. If the link is removed, there would be no way for keyboard users to scroll through tip content.

**Comment by driskull** [comment]
Yeah, we can add a tabIndex to it.

**Comment by driskull** [comment]
Added!

**Comment by annelfitz** [comment]
Thanks @driskull that works great!

A couple things: 
  - I noticed that in Safari, the link inside of the tip is no longer accessed by tabbing. All other browsers it tabs from the 'X', to the tip content, to the link, then to the navigation arrows. Safari skips the link
 - This might be me being picky, but the focus outline of the tip content gets caught under the 'X' button of the tip manager (only in Chrome and Safari). I've noticed this throughout the calcite components though so I understand if this is not a priority
![Screen Shot 2019-10-24 at 9 16 14 AM](https://user-images.githubusercontent.com/51344676/67505177-a0a43780-f63f-11e9-9726-0571ff4dae58.png)
